pojo/src/main/java/com/connexta/transformation/pojo/unknown/UnknownMetadataPojo.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.transformation.pojo.unknown;
+
+import com.connexta.transformation.pojo.MetadataPojo;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+
+/**
+ * This class provides a pojo implementation for unknown request information objects and is used for
+ * forward compatibility where the current code might not be able to understand a new type of
+ * resource information. In such case, Jackson would map this new type of request information object
```

Comment:
✏️ change `request information` to metadata.

Reply:
cut and paste 😄